<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_Dialog</class>
 <widget class="QDialog" name="main_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>931</width>
     <height>121</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>140</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>941</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_read">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>读写测试</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="group_setting">
         <property name="title">
          <string>settings</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>SlaveAddr</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_slv_addr">
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>StartAddr</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_reg_addr">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="singleStep">
               <number>0</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Number of Register</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_len">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>32</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_read"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btn_read">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_write">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>样例</string>
      </property>
      <widget class="QPushButton" name="btn_6axis">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>设置为6轴模式</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_save">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>保存参数到Flash</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_rst">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_9xias">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>设置为9轴模式</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_read_param">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>读取校准参数</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_param">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>321</width>
         <height>111</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
